composer.json

```diff
@@ -3,20 +3,14 @@
     "description": "Print info from db-extractor",
     "type": "library",
     "license": "MIT",
-    "authors": [
-        {
-            "name": "Ondra Jodas",
-            "email": "ondra.jodas@gmail.com"
-        }
-    ],
     "require": {
         "monolog/monolog": "^1.24",
         "keboola/csv": "^1.1"
     },
     "require-dev": {
-        "keboola/coding-standard": "^8.0",
```

Comment:
sorry, tady zas není důvod to ponižovat z v 8 na 2 dtto phpstan a phpstan-shim má být

Reply:
vrátil jsem CS na v8 a místo phpstan použil phpstan-shim